apps/src/templates/teacherNavigation/teacher-navigation.module.scss

```diff
@@ -75,7 +83,7 @@
   margin-bottom: 2px;
   padding-left: 16px;
   padding-top: 20px;
-  color: $neutral_dark80;
+  color: #727a83;
```

Comment:
Does this color have a name attached to it in Figma? Maybe worth adding it to the shared colors file if so.

Reply:
This was intentional in part because it was in the figma but not in the DSCO files so we had this in the backlog of items to change in the future... BUT, I checked in with Denys and it looks like he had just updated the DSCO files and it is now in there.  I am making this change now!